--- mojombo-grit-7609e74-o/lib/grit/git-ruby/repository.rb	2010-08-19 18:55:45.000000000 -0600
+++ mojombo-grit-7609e74/lib/grit/git-ruby/repository.rb	2010-08-28 11:26:32.000000000 -0600
@@ -13,7 +13,11 @@
 require 'grit/git-ruby/internal/loose'
 require 'grit/git-ruby/git_object'
 
-require 'rubygems'
+begin
+  require 'rubygems'
+rescue LoadError
+end
+
 require 'diff/lcs'
 require 'diff/lcs/hunk'
 
--- grit-2.4.1/lib/grit.rb~	2013-05-05 17:53:11.000000000 +0300
+++ grit-2.4.1/lib/grit.rb	2013-05-05 17:56:31.533333692 +0300
@@ -13,14 +13,16 @@
 
 begin
   require 'mime/types'
-  require 'rubygems'
 rescue LoadError
-  require 'rubygems'
   begin
-    gem "mime-types", ">=0"
-    require 'mime/types'
-  rescue Gem::LoadError => e
-    puts "WARNING: Gem LoadError: #{e.message}"
+    require 'rubygems'
+    begin
+      gem "mime-types", ">=0"
+      require 'mime/types'
+    rescue Gem::LoadError => e
+      puts "WARNING: Gem LoadError: #{e.message}"
+    end
+  rescue LoadError
   end
 end
 
