Only in mojombo-grit-7609e74: InstalledFiles
diff -ur mojombo-grit-7609e74-o/lib/grit/git-ruby/repository.rb mojombo-grit-7609e74/lib/grit/git-ruby/repository.rb
--- mojombo-grit-7609e74-o/lib/grit/git-ruby/repository.rb	2010-08-19 18:55:45.000000000 -0600
+++ mojombo-grit-7609e74/lib/grit/git-ruby/repository.rb	2010-08-28 11:26:32.000000000 -0600
@@ -13,7 +13,11 @@
 require 'grit/git-ruby/internal/loose'
 require 'grit/git-ruby/git_object'
 
-require 'rubygems'
+begin
+  require 'rubygems'
+rescue LoadError
+end
+
 require 'diff/lcs'
 require 'diff/lcs/hunk'
 
diff -ur mojombo-grit-7609e74-o/lib/grit.rb mojombo-grit-7609e74/lib/grit.rb
--- mojombo-grit-7609e74-o/lib/grit.rb	2010-08-19 18:55:45.000000000 -0600
+++ mojombo-grit-7609e74/lib/grit.rb	2010-08-28 11:25:09.000000000 -0600
@@ -18,12 +18,15 @@
 end
 
 # third party
-require 'rubygems'
 begin
-  gem "mime-types", ">=0"
-  require 'mime/types'
-rescue Gem::LoadError => e
-  puts "WARNING: Gem LoadError: #{e.message}"
+  require 'rubygems'
+  begin
+    gem "mime-types", ">=0"
+    require 'mime/types'
+  rescue Gem::LoadError => e
+    puts "WARNING: Gem LoadError: #{e.message}"
+  end
+rescue LoadError
 end
 
 # ruby 1.9 compatibility
